crates/ty_python_semantic/src/types/property_tests/type_generation.rs

```diff
@@ -1,8 +1,9 @@
 use crate::db::tests::TestDb;
 use crate::place::{builtins_symbol, known_module_symbol};
+use crate::types::tuple::TupleType;
```

Comment:
Nit: Should we re-export this type from `types` to reduce some of those import-only diffs?